_docs/_latest/use/config.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Config
+order: 007
+---
+
+# {{ page.title }}
+
+Forseti Config is a tool for you to set the configuration of the CLI.
```

Comment:
nit: instead of a `tool`, describe it as `option` or `command`, since this is really not a separate tool.  Or leave it out all together: ```Forseti Config lets you set the configuration of the CLI```.